lib/inspec/base_cli.rb

```diff
@@ -364,15 +364,15 @@ def configure_logger(o)
       o[:logger].level = get_log_level(o.log_level)
     end
 
-    def mark_text(text)
+    def self.mark_text(text)
```

Comment:
OK, I think what you want is "I want my base CLI class to help me with formatting".  Make these public, instance methods, not "attempted private" class methods. 

Then, have ProfileGenerator inherit from here, or we make a real UI object.